README.md

```diff
@@ -1,5 +1,6 @@
 # Wasm-shim
 
+[![Rust](https://github.com/Kuadrant/wasm-shim/actions/workflows/rust.yaml/badge.svg)](https://github.com/Kuadrant/wasm-shim/actions/workflows/rust.yaml)
```

Comment:
this was added to test new runs of license scans, but good to have as well, so I leave it